gensim/models/keyedvectors.py

```diff
@@ -1385,6 +1385,36 @@ def init_sims(self, replace=False):
             logger.info("precomputing L2-norms of word weight vectors")
             self.vectors_norm = _l2_norm(self.vectors, replace=replace)
 
+    def relative_cosine_similarity(self, wa, wb, topn=10):
+        """Compute the relative cosine similarity between two words given top-n similar words,
+        by Artuur Leeuwenberg, ... "A Minimally Supervised Approach for Synonym Extraction with Word Embeddings"
```

Comment:
should be proper link rendered by sphinx, in this case

```
by `Artuur ..... <https://ufal.mff.cuni.cz/pbml/105/art-leeuwenberg-et-al.pdf>`_
```

Reply:
Done. :+1:

Reply:
Do I have to include another authors and paper name in < >?

Reply:
just replace with this
```
`Artuur Leeuwenberga, Mihaela Velab , Jon Dehdaribc, Josef van Genabithbc "A Minimally Supervised Approach for Synonym Extraction
with Word Embeddings" <https://ufal.mff.cuni.cz/pbml/105/art-leeuwenberg-et-al.pdf>`_.
```

Reply:
Thanks :smile: